src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -66,6 +66,7 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
     messageEntity.setProperty("recipient", message.getRecipient());
+    messageEntity.setProperty("sentimentScore", message.getSentimentScore());
```

Comment:
Yeah I was confused as to why that was happening because I had created a separate branch. When you say 'start from the master branch' does that mean I should switch to the master branch first and then create a new branch?

Reply:
Yes, that is correct. When you create a new branch, it persists all the context of the branch that you are currently on (so if there are any commits in the current branch, those will also be copied over to the new branch that you create). So, to get a clean branch, one should always start new branches by first switching to the master branch before creating the branch.

Reply:
Ahh I see, I didn't know that. Thanks for clarifying!